src/hooks/useDebounce.js

```diff
@@ -0,0 +1,16 @@
+import {useEffect, useRef} from 'react';
+import _ from 'underscore';
+
+export default function useDebounce(func, wait, immediate) {
```

Comment:
A simple JSDoc? Doesn't have to define the concept of debouncing, but can "steal" the meaning of the parameters from the relevant `underscore`'s JSDoc

Reply:
(or Lodash JSDoc)